pypeit_files/keck_deimos_830g_l_8100.pypeit

```diff
@@ -10,6 +10,9 @@
 [reduce]
     [[findobj]]
         maxnumber = 5
```

Comment:
Given the DEIMOS plate scale of 0.1185, I suspect a bunch of defaults should be set to this larger value??

Reply:
I am not sure I follow here.
This is set to test that `use_user_fwhm` works properly. The final `opt_fwhm` for this object is around 1 arcsec (when we let PypeIt to compute it). Therefore I set  `find_fwhm = 9.` (9*0.1185~1") and stop PypeIt from computing the fwhm (with `use_user_fwhm = True`), so that the final `opt_fwhm` for this object will still be ~1 arcsec.

Reply:
Yeah I have not checked the deimos default for fwhm, but I doubt it is 9 pixels. That was my point.